free-1.md

```diff
@@ -10,34 +10,48 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+My idea would help physics students (or just people in general) do simulations involving some basic forces (like gravity and magnetic forces). As of right now, many physics students don't have very good access to simple simulators that they can play around with. There are some online, but they're written in Java and are very difficult to get working. This DSL would hopefully make it easy to set up simple demos and create animations of the results of the simulaiton.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Some things are best learned through demonstration or experimentation, but not every concept in physics is easy to safely demonstrate or play with. This DSL would help make very small or very large simulations accessible to physics students, which could help them with understanding concepts they encounter in class.
+
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+Many years ago I actually worked on a simulation program as part of my research at the time. I also have some friends who are physics majors too. This could be another opportunity to work on such a simulation program and actually work with people who might actually use it for help / feedback.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Simple physics simulations.
+
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user would likely describe objects and their initial conditions (like velocity) in a file. This could be done in a variety of different ways, including some that we discussed in class. "Programming" would involved setting up a simulation with everything in the right place. Depending on the implementation, numerical limits may have to be considered when writing a program as it's really easy to for simulations to become unstable unless the simulation software is written well.
+
+This language could theoretically be interpreted too. So, objects could be added or removed when the simulation is paused and the user could control how long the simulation would run before pausing again.
```

Comment:
This is an interesting idea - it's worth considering whether a compiled or interpreted interface is what your users would prefer. You could likely provide both as well. I think you'd probably get a lot of milage out of both of these.